hera_pspec/pspecdata.py

```diff
@@ -1042,14 +1119,18 @@ def set_spw(self, spw_range, ndlys=None):
         ndlys : integer
             Number of delay bins. Default: None, sets number of delay
             bins equal to the number of frequency channels in the spw.
+        set_Ndlys : bool, optional
```

Comment:
I've added more context. Short answer, sometimes we need to broadcast flags for empirical error bars and when we do this, we don't want n_spw to be changed to nfreqs if they were not equal in the first place. 

Long answer: Because, if our data contains foregrounds in it, but different flagging patters for each time, then an empirical covariance, estimated from the data waterfall, will have lots of (small but significant) artificial jumps in it -- spectral structure. These spectral structures will cause our estimates of the error bars on p-hat to be contaminated heavily be foregrounds and since we only have many flagging patterns but compute error bars with tr(E_1 \hat{C} E_2 \hat{C}) for one effective time. The E-matrices in the formulas for p-hat error bars remove this contamination only if the flags in \hat{C} match the flags in E. Thus, to avoid contamination at all delays from foregrounds in empirical error-bars, one actually does need consistent flags in time so that the E-matrices in tr(E_alpha C E_beta C) (and other formulas in `get_unormmed_V` )can perform the appropriate foreground subtraction to avoid contaminated error bar estimates.